localstack/services/awslambda/lambda_api.py

```diff
@@ -819,7 +819,10 @@ def execute(event, context):
             lambda_handler = execute
 
         if runtime.startswith("go1") and not use_docker():
-            install_go_lambda_runtime()
+            go_installer = awslambda_go_runtime_package.get_installer()
+            if not go_installer.is_installed():
+                go_installer.install()
+
```

Comment:
You can just call `awslambda_go_runtime_package.install()` here.

Reply:
:+1: